mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
@facelessuser perhaps the code would be clearer if the `construct_yaml_str` function was defined before the `Loader` subclass (jsut reorder them in the source). It is easy to miss that it is not indented and not a class method (especially as its first argument is `self`). Reordering might help eliminate that confusion for future readers.